docs/resources/core/console_history.md

```diff
@@ -1,29 +1,33 @@
- # oci\_core\_console_history
+# oci\_core\_console_history
+
+[ConsoleHistory Reference][03f57b12]
+
+  [03f57b12]: https://docs.us-phoenix-1.oraclecloud.com/api/#/en/iaas/20160918/ConsoleHistory/ "ConsoleHistoryReference"
```

Comment:
Hi Dan, I'm unfamiliar with this linking structure, can you describe how it works?